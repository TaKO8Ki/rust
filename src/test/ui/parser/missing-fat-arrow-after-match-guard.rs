fn main() {
    let x = 1;
    let y = 2;
    let value = 3;

    match value {
        Some(x) if x == y {
            //~^ ERROR expected one of `!`, `.`, `::`, `=>`, `?`, or an operator, found `{`
            x
        },
        _ => {
            y
        }
    }
}
