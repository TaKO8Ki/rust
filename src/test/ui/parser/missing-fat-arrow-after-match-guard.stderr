error: expected one of `!`, `.`, `::`, `=>`, `?`, or an operator, found `{`
  --> $DIR/missing-fat-arrow-after-match-guard.rs:7:27
   |
LL |         Some(x) if x == y {
   |                           ^ expected one of `!`, `.`, `::`, `=>`, `?`, or an operator
   |
help: try adding a fat arrow here
   |
LL |         Some(x) if x == y => {
   |                           ++

error: aborting due to previous error

